Install "file"

Raizo62 opened the pull request:
Hi

- PCredz uses the "file" command, but the "file" package which includes this command is not a dependency of other installed packages
- Readme.md : replace "apt-get" by "apt" to be consistent

mjbear commented:
> Hi
> 
>     * PCredz uses the "file" command, but the "file" package which includes this command is not a dependency of other installed packages
> 
>     * Readme.md : replace "apt-get" by "apt" to be consistent

@Raizo62
I agree with standardizing apt vs apt-get.

Could you elaborate or confirm this "file" issue still occurs?

From the Docker image I built today I have the `file` command available so it would stand that the `file` command is part of the `python:3.7.15-slim-buster` base image.

```bash
PCredz]$ docker run --rm -it pcredz
root@94df4c3057d5:/opt/Pcredz# file
file      filefrag  
root@94df4c3057d5:/opt/Pcredz# file
```

Raizo62 commented:
The command file is in the python:3.7.15-slim-buster base image.

But, if we install PCredz on a minimal system (a minimal docker image or a minimal OS), the command is not present.  I got the error with a docker image of Parrot OS.
So I added the "file" package in the list of necessary files.

To be consistent, I added it in the dockerfile. And since it is already present, it does not change anything (no error)

mjbear commented:
> But, if we install PCredz on a minimal system (a minimal docker image or a minimal OS), the command is not present. I got the error with a docker image of Parrot OS. So I added the `file` package in the list of necessary files.

Thank you for the clarity. Understood. :grin:

> To be consistent, I added it in the dockerfile. And since it is already present, it does not change anything (no error)

Right, if it's present on the base image already then the Debian package manager just wouldn't have anything to install. :+1:

lgandx commented:
Merged!
